### ISSUE TITLE
iOS/iPadOS app crashes when starting sync

### ISSUE THREAD

**Issue by Machiel692** [osmandapp/OsmAnd-iOS#2999]
Every time when I try to sync (this is a subscribe service!!) the app crashes. I have already sent the logfile a while ago but nothing changed sofar (no app updates!). I have almost 500 favourites to sync! Don't think I should attach the logfile here in public (because of privacy reasons).

**Comment by DmitryAlexei** [comment]
Please add a short video or description of what you are doing before this bug occurs.
you can also send your logs to support@osmand.net

**Comment by DmitryAlexei** [comment]
Probably related to https://github.com/osmandapp/OsmAnd-iOS/issues/2988

**Comment by Machiel692** [comment]
> Please add a short video or description of what you are doing before this bug occurs.
> you can also send your logs to [support@osmand.net](mailto:support@osmand.net)

I had send my logs already to [support@osmand.net](mailto:support@osmand.net) some weeks ago. Nevertheless, this morning I tried to sync again and on iOS it did not immediately crash the app as before (immediately at 1% of sync) but it performed a full sync until 100%, then it stays at this 100% for a very long time. But at least I noticed that the favourites have also synced through to my Mac app. On my Mac the syncing was painfully slow but in the end it worked. I will keep my subscription!
